shell/src/main/java/alluxio/cli/ValidateEnv.java

```diff
@@ -81,26 +84,36 @@
       "workers.ssh.reachable",
       new SshValidationTask("workers"));
 
+  // space validations
+  private static final ValidationTask WORKER_RAMDISK_SPACE_VALIDATION_TASK = registerTask(
+      "worker.ramdisk.space",
+      new RamDiskSpaceValidationTask());
+
   private static final Map<String, Collection<ValidationTask>> TARGET_TASKS =
       initializeTargetTasks();
 
   private static Map<String, Collection<ValidationTask>> initializeTargetTasks() {
     Map<String, Collection<ValidationTask>> targetMap = new TreeMap<>();
-    targetMap.put("master", Arrays.asList(
-        MASTER_RPC_VALIDATION_TASK,
-        MASTER_WEB_VALIDATION_TASK,
-        PROXY_WEB_VALIDATION_TASK,
+    ValidationTask[] commonTasks = {
         MASTERS_SSH_VALIDATION_TASK,
+        PROXY_WEB_VALIDATION_TASK,
         WORKERS_SSH_VALIDATION_TASK
```

Comment:
I have question about these common tasks. Why would a master node need to run a `WORKERS_SSH_VALIDATION_TASK` and why would a worker node need to run a `MASTERS_SSH_VALIDATION_TASK`?